openxla-nvgpu/runtime/src/openxla/runtime/nvgpu/cudnn_module.cpp

```diff
@@ -23,12 +29,31 @@ using namespace iree;
 
 class CuDNNModuleState {
```

Comment:
I think you want to make this class non-movable, non-copyable.

Reply:
Done.